Makefile

```
@@ -1,7 +1,7 @@
 .PHONY: build clean install
 
 BINARY_NAME=browsir
-INSTALL_PATH=/usr/local/bin
+INSTALL_PATH=$(HOME)/bin
```

It is likely that not everyone has the `bin` directory in the home so in case it would have to be created during installation.

@404answernotfound I think the best unix/linux like directory to put binary like this is `$HOME/.local/bin` also this is recommended by the XDG Base Directory Specification for storing user-specific executable files, see [here](https://specifications.freedesktop.org/basedir-spec/latest/#variables)

---

my 2cents on this: looking at how others are doing e.g. docker, docker-compose, kubectl, npm, npx I'd say it'll be better to have it by default on `/usr/local/bin`.

Later you could create an issue to handle the $HOME local specific directory for the package if the user is answering on a future prompt question to install the package there, else it'll be installed directly on the /usr/local/bin

---

It depends on how we want to handle or use browsir, `/usr/local/bin` is system wide and `$HOME/.local/bin` is specific for the user.

Most of the time the `/usr/local/bin` directory is been used by the package manager of the system. Other manager like versions manager can also install in others path, e.g. I use [asdf](https://asdf-vm.com/) and my nvim binary is on `$HOME/.asdf/shims/nvim`. Also there are `/opt` (unix directory for **optional**) that is been use to put projects, binaries and so on, to use like a workspace area.

If we have to add browsir to a central package repository (like AUR) we can install to `/usr/local/bin`, also homebrew is not using `/usr/local/bin` as default, but it use `/opt/homebrew`.

For me if the user clone the repository and install by himself the right path is `$HOME/.local/bin` on the other hand `/usr/local/bin` can be use for package manager installation setup.

---

We could also think to expand the variable in order to let it be overwritable, like this:

```suggestion
INSTALL_PATH ?= $(HOME)/.local/bin/
```